docs/user/reference/analytics.rst

```diff
@@ -0,0 +1,38 @@
+Analytics for Search and Traffic
+================================
+
+Read the Docs supports analytics for search and traffic.
+When someone reads your documentation,
+we collect data about the vist and the referer
+with full respect of the privacy of the visitor.
+
+
+Traffic Analytics
+-----------------
+
+Read the Docs aggregates simple statistics about visits to your documentation.
+
+Traffic Analytics lets you see *which* documents your users are reading.
+This allows you to understand how your documentation is being used,
+so you can focus on expanding and updating parts people are reading most.
+
+If you require more detailed analytics, Read the Docs has native support for Google Analytics.
+It's also possible to customize your documentation to include other analytics frameworks.
+
+Learn more in :doc:`/analytics`.
+
+
+Search Analytics
+----------------
+
+When someone visits your documentation using a search engine,
+Read the Docs will collect analytics on their search term.
```

Comment:
This is about in-doc search, not incoming search terms. Shows how little we've been doing a good job of documenting it.  😆

Reply:
:facepalm: 

I didn't continue writing for the exact reason of feeling like something was terribly wrong :D

Reply:
This has been clarified and a new how-to for search analytics has been created :+1: